src/config.js

```diff
@@ -20,8 +20,9 @@ export const chartMinWidth = 1200;
 
 // Remember to update the 'Flags' section in the README when updating these:
 export const flags = {
-  newgraph: {
-    description: 'Improved graphing algorithm',
+  oldgraph: {
+    description:
+      'display older version of graph without improved graphing algorithm',
```

Comment:
For consistency (as this is displayed to users), can you please capitalise the first letter of this string?

Also I would probably shorten it. Something like this?

```suggestion
      'Use older Dagre graphing algorithm',
```

Reply:
Thanks for the recommendation - this is great and I've literally just used that :D